internal/builder/builder.go

```diff
@@ -567,45 +547,31 @@ func (b *Builder) rootOwnedDir(path string, time time.Time) *tar.Header {
 	}
 }
 
-func (b *Builder) orderLayer(order dist.Order, dest string) (string, error) {
-	contents, err := orderFileContents(order)
+func (b *Builder) lifecycleLayer(dest string) (string, error) {
```

Comment:
I didn't change these methods at all - all I did was reorder them in order of usage, to simplify the structure of the file.